packages/main/test/sap/ui/webcomponents/main/samples/Input.sample.html

```diff
@@ -140,6 +151,42 @@ <h3>Input with Value State</h3>
 		</xmp></pre>
 	</section>
 
+	<section>
+		<h3>Input as Search Field</h3>
+		<div class="snippet">
+			<div class="flex-column samples-margin">
+				<ui5-input id="searchInput" placeholder="Enter search criteria ...">
+					<ui5-icon id="searchIcon" data-ui5-slot="icon" src="sap-icon://search" class="inputIcon"></ui5-icon>
+				</ui5-input>
+			</div>
+		</div>
+		<pre class="prettyprint lang-html"><xmp>
+<style>
+	.inputIcon {
+		width: 2.375rem;
+		background: #fafafa;
+		color: #0a6ed1;
+		cursor: pointer;
+	}
+	.inputIcon:hover{
+		background: rgba(224,224,224,0.5)
+	}
+</style>
+
+<ui5-input id="searchInput" placeholder="Enter search criteria ...">
+	<ui5-icon id="searchIcon" data-ui5-slot="icon" src="sap-icon://search" class="inputIcon"></ui5-icon>
+</ui5-input>
+
+var searchCriteria = "PASTA";
+searchIcon.addEventListener("press", function(){
+	alert("Look for: " + searchCriteria);
+});
+searchInput.addEventListener("change", function(e){
+	searchCriteria = e.target.value;
+});
```

Comment:
Add script tag at the beginning and the end of code snippet

Reply:
I added the script tags